src/test_workflow/bwc_test/bwc_test_suite_opensearch.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+
+from test_workflow.bwc_test.bwc_test_suite import BwcTestSuite
+
+
+class BwcTestSuiteOpenSearch(BwcTestSuite):
+
+    def __init__(
+        self,
+        work_dir,
+        component,
+        test_config,
+        test_recorder,
+        manifest
+    ):
+
+        super().__init__(
+            work_dir,
+            component,
+            test_config,
+            test_recorder,
+            manifest
+        )
+
+    def get_cmd(self, script, security, manifest_build_location):
+        return f"{script}"
```

Comment:
This should be updated but current `bwctest.sh` doesn't accept other arguments: https://github.com/opensearch-project/anomaly-detection/blob/main/bwctest.sh